src/daemon/absorber.cc

```diff
@@ -71,15 +85,19 @@ bool Absorber::HandleNewMessage(net::ConnectionPtr connection,
     Message execute(message->ReleaseExtension(proto::Remote::extension));
     DCHECK(!execute->flags().compiler().has_path());
     if (execute->has_source()) {
-      if (tasks_->Push(Task{connection, std::move(execute)})) {
-        return true;
+      Task task{connection, std::move(execute), HandledHash()};
```

Comment:
Put some comment that we plan in the future to take handled hash from incoming message.